crdb-versions.yaml

```diff
@@ -46,4 +45,8 @@ CrdbVersions:
   - v21.1.0
   - v21.1.1
   - v21.1.2
+  - v21.1.3
```

Comment:
Didn't we pull this version? Why is it still listed here? Same with `21.1.4` no?

Reply:
We pulled it, but it's still accessible. Probably worth keeping it here, so we support migration from those versions?